README.md

```diff
@@ -0,0 +1,123 @@
+# mcti-detect
+
+[![Build Status](https://travis-ci.org/krono-safe/mcti-detect.svg?branch=master)](https://travis-ci.org/krono-safe/mcti-detect)
+
+Multi-Core Time-Interferences Detector (or `mcti-detect`) is a program written
+using the [D programming language][1] that analyses a **time-constrained
+application** and checks whether the provided exclusion groups hold or not.
+
+It is a proof-of-concept for the algorithms described in the paper (to be
+published) **Preventing Multi-Core Time-Interferences by Design**.
+
+## Building from sources
+
+You first need to install a [D toolchain][2] for your specific platform.  Then,
```

Comment:
As a matter of fact, you need the latest version of _DMD_, or so it seems.
I couldn't compile the application with gdc-8 for instance.

Reply:
Okay... well, since it may change in the future, I will leave it as is. Travis can be used to check what versions of the D compiler actually work.